doc/man/8/ceph-fuse.rst

```diff
@@ -39,7 +39,7 @@ Any options not recognized by ceph-fuse will be passed on to libfuse.
 
 .. option:: -d
 
-   Detach from console and daemonize after startup.
+   Debug mode: like -f, but also send all log output to stderr.
```

Comment:
I'd prefer: `run in foreground, and send all log otuput to stderr and enable FUSE debugging (-o debug).`